app/ts-monitor/run/server.go

```diff
@@ -71,7 +75,13 @@ func NewServer(conf config.Config, cmd *cobra.Command, logger *logger.Logger) (a
 			time.Sleep(time.Second)
 		}
 	}()
-	go func() { _ = http.ListenAndServe("127.0.0.1:6066", nil) }()
+	http.HandleFunc("/metrics", serveMetrics)
+	go func() {
+		err := http.ListenAndServe("127.0.0.1:6066", nil)
```

Comment:
`err := http.ListenAndServe("127.0.0.1:6066", nil)`

Here， we should get the listen address from the config file as much as possible, not 127.0.0.1. Because someone can access it from another machine

Reply:
I have modified to read from the configuration

Reply:
Now the problem has been solved.